packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -187,6 +187,16 @@ export type AzureCommunicationChatAdapterArgs = {
     threadId: string;
 };
 
+// @public
+export interface BackgroundImage {
+    extension?: string;
```

Comment:
do we need `extension`?

Reply:
We don't need extension as of now, but if we are planning to restrict few extensions, we can use extension to filter the backgrounds before adding to the background set. Any thoughts on that?

Reply:
I would say remove this, if we get an image with an unsupported extension we'll simply have to throw an error (well we'll propagate the underlying error that the calling sdk will throw)

Reply:
I like the idea of catching this at compile time -- e.g. we could strongly type this to be:
```suggestion
    extension?: 'png' | 'jpeg' | 'jpg' | etc...;
```

But I don't want Calling SDK to suddenly add support for new types at the service level and we have arbitrarily limited what can be accepted.
I think we have to follow Calling's API on this and simply take any url and try to render it